src/backend/gporca/libgpopt/include/gpopt/metadata/CTableDescriptor.h

```diff
@@ -54,6 +54,9 @@ class CTableDescriptor : public CRefCount,
 	// name of table
 	CName m_name;
 
+	// table alias
+	CName *m_alias;
```

Comment:
How about changing the type of this field to CWStringConst? Then below, in the SetAlias method there will be no need for a constructor, and in the destructor there will be no need for deletion and nulling.

Reply:
m_alias is CName to be consistent with declaration of m_name above. Pointer is needed to make it optional. Changing it to CWStringConst pointer will result in using similar constructor and destructor and using an instance will require additional boolean optional flag and more complicated logic.